### DISCUSSION TITLE
Add `Save Custom Item` feature

### DISCUSSION THREAD

**Discussion by BachiMjavanadze** [cyrilletuzi/vscode-angular-schematics#909] [Ideas]
Please add the ability to save my Custom Items. Let's say I very often create a component with an inline style and an inline template. I don't want to go into settings every time. Just add another button:

![save custom item](https://github.com/cyrilletuzi/vscode-angular-schematics/assets/38082501/82fc646e-fd78-4f31-b6b4-57ebe12456aa)

and after that the following should appear in the list of components:

![my items](https://github.com/cyrilletuzi/vscode-angular-schematics/assets/38082501/1af2b62a-6cad-45a4-bb23-5e9fea732a46)

P.S. Of course, Custom Items must sync across devices. If I enter my credentials on the new computer in the newly installed VSCode, all my custom items should be retrieved.

**Comment by cyrilletuzi** [comment]
Hi @BachiMjavanadze, adding custom schematics is available in the Pro edition. You will find all the details on the [dedicated website](https://cyrilletuzi.gumroad.com/l/schematicspro/).

> P.S. Of course, Custom Items must sync across devices. If I enter my credentials on the new computer in the newly installed VSCode, all my custom items should be retrieved.

Just note that custom schematics are registered directly in the project, because one of the goal is to share common good practices across the whole team working on it. But registering schematics at a user level could be a feature to add if there is demand.
